sdk/src/main/java/com/hedera/hashgraph/sdk/Client.java

```diff
@@ -144,6 +144,27 @@ static ExecutorService createExecutor() {
             threadFactory, new ThreadPoolExecutor.CallerRunsPolicy());
     }
 
+    /**
+     *
+     * Construct a client given a set of nodes.
+     * It is the responsibility of the caller to ensure that all nodes in the map are part of the
+     * same Hedera network. Failure to do so will result in undefined behavior.
+     * The client will load balance all requests to Hedera using a simple round-robin scheme to
+     * chose nodes to send transactions to. For one transaction, at most 1/3 of the nodes will be tried.
+     *
+     * @param networkMap the map of node IDs to node addresses that make up the network.
+     * @param executor runs the grpc requests asynchronously. Note that calling `close()` method on one of the
+     *                 clients will close the executor for all the other clients sharing this executor
+     * @return {@link com.hedera.hashgraph.sdk.Client}
+     */
+    public static Client forNetworkWithExecutor(Map<String, AccountId> networkMap, ExecutorService executor) {
```

Comment:
Changing name seems redundant. Why not overload the existing method?
```suggestion
    public static Client forNetwork(Map<String, AccountId> networkMap, ExecutorService executor) {
```
Ditto for other methods below.

Reply:
Done